README.md

```diff
@@ -3,8 +3,7 @@
 # Library and tool for running OCI-based containers in Pods
 
 Libpod provides a library for applications looking to use the Container Pod concept,
-popularized by Kubernetes.  libpod also contains the `podman` tool, for managing
-Pods, Containers, and Container Images.
+popularized by Kubernetes.  libpod also contains the Pod Manager tool `(podman)`, Podman manages Pods, Containers, and Container Images, and Container Volumes.
```

Comment:
`Podman` vs. `podman`, `libpod` vs. `Libpod`.

I think that `Podman manages Pods [...]` is a new sentence.